main.cpp

```diff
@@ -297,16 +366,30 @@ struct School
 
     };
 
-
-    //educate children
-    void educateChildren();
-    //employ adults
-    void employAdults(std::string Over21Yo);
-    //play sports
-    bool playSports(bool notRaining = true);
+    void educateChildren(bool amountChildren);
+    void employAdults(int Over21Yo);
+    void playSports(bool notRaining);
 
 };
 
+void School::educateChildren(bool amountChildren)
+{
+    amountChildren = true;    
+}
+
+void School::employAdults(int Over21Yo)
+{
+    if (Over21Yo > 21)
+    {
+        Over21Yo = 22;
+    }  
+}
+
+void School::playSports(bool notRaining)
+{
+    notRaining = true; 
+}
```

Comment:
Fix this according to my previous pull request review instructions
- [ ] revise